ts/node/sql.ts

```diff
@@ -646,10 +647,23 @@ function getAllOpenGroupV2ConversationsIds(): Array<string> {
 }
 
 function getPubkeysInPublicConversation(conversationId: string) {
+  const conversation = getV2OpenGroupRoom(conversationId);
+  if (!conversation) {
+    return [];
+  }
+
+  const hasBlindOn = Boolean(
+    conversation.capabilities &&
```

Comment:
Just double checked with lodash and you don't need this line. If it's undefined isArray will through false. 😄

Reply:
Disregard this.